packages/openactive-integration-tests/test/helpers/flow-helper.js

```diff
@@ -79,9 +87,40 @@ class FlowHelper {
     return this.state.putOrder();
   }, { cachePromiseRejection: true });
 
+  P = pMemoize(async () => {
+    this._assertStageShouldNotBeSkipped('P');
+    this.getOrderAfterPPromise = this.state.getOrderAfterP();
```

Comment:
This also makes the assumption that P is always followed by a `getOrder` check, which should not be the case as P does not add anything to the Orders Feed ("A" does, but which is simulated by the test interface action)